source/includes/document-filtering-limitations.rst

```diff
@@ -0,0 +1,27 @@
+- ``field`` names can't contain dots (``.``) or dollar signs (``$``).
+
+- ``matchValues`` array items can't be any of the following BSON types: 
+
+  - Arrays 
+  - Embedded documents 
+  - Regular expressions
+
+- Documents must not move in or out of the filter during migration. If a source 
+  document from a prior sync no longer matches the query filter, 
+  ``mongosync`` deletes the document from the destination cluster.
+
+- If the document filter contains a string constant and at least one 
+  migrated collection that is specially collated, ``mongosync`` fails 
```

Comment:
It might be good to contextualize “specially collated”? I don’t know how widely-used that term is.

Reply:
I can adjust the language to instead say "non-default" collation, especially given that "non-default collation" has been used throughout our docs before. LMK if this is still unclear and/or inaccurate.